### DISCUSSION TITLE
Dock Issues

### DISCUSSION THREAD

**Discussion by bjacobs39** [elementary/triage#607] [Q&A]
Hi. Downloaded the early access iso. The dock overlap is an issue, if I have an app open maximized then I can't click near the bottom of the app were it overlaps with the dock, opens the app in the dock.  If I hover  the mouse towards the bottom it does not show the dock but does bring up the text of the app in the dock. Tries different options only way it works correctly is if I show the dock rather than hiding it. I will mention I am running it as a virtual to test so not sure if that may be the issue.

**Comment by lenemter** [comment]
Thanks for reporting this. This is a duplicate of https://github.com/elementary/gala/issues/2101.
